apps/new-dealer/behaviours/renew-vary-warning.js

```diff
@@ -0,0 +1,11 @@
+module.exports =
+  superclass => class extends superclass {
+    configure(req, res, next) {
+      if (req.sessionModel.get('activity') === 'renew' || req.sessionModel.get('activity') === 'vary') {
+        Object.keys(req.form.options.fields).forEach(key => {
+          req.form.options.fields[key].isWarning = true;
+        });
+      }
+      next();
+    }
+  };
```

Comment:
So we need a custom behaviour in our apps for a warning? Maybe in the future it be good to build into hof?

Reply:
The reason the custom behaviour was needed here was because of the fork dependent on whether the user was varying/renewing the application. The warning was only needed in those cases and not for a new application.  This was specific to firearms which is why I haven’t added a behaviour to hof at this time. In any other case, a warning could be included in a page by adding the "isWarning : true" config in fields which I have built in hof.

Reply:
Ah, that's sensible 👍

Reply:
can add a comment about it what you said in the code?

Reply:
I have added in a comment now.